app/scripts/directives/istagSelect.js

```diff
@@ -35,9 +35,7 @@ angular.module("openshiftConsole")
         $scope.isNamesByNamespace = {};
 
         // Check if the istag object contains data about namespace/imageStream/tag so the ui-select will be pre-populated with them
-        var shouldPrepopulate = _.has($scope, 'istag.namespace') &&
-                                _.has($scope, 'istag.imageStream') &&
-                                _.has($scope, 'istag.tagObject.tag');
+        var shouldPrepopulate = (_.get($scope, 'istag.namespace') && _.get($scope, 'istag.imageStream') && _.get($scope, 'istag.tagObject.tag')) ? true : false;
```

Comment:
@jhadvig I find this harder to understand than before. Generally you shouldn't have

```js
var foo = <expression> ? true : false;
```

It should usually just be

```js
var foo = <expression>;
```

I'd keep it like it was before, but replacing `_.has` with `_.get`.